base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -1894,3 +1914,54 @@ fn unwrap_or_clone<T: Clone>(arc: Arc<T>) -> T {
         .or_else(|t| Result::<_, Infallible>::Ok(Clone::clone(&*t)))
         .expect("or_else with Ok")
 }
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::test_helpers::blockchain::create_store;
+    use tari_test_utils::paths::create_temporary_data_path;
+    #[test]
+    fn lmdb_fetch_monero_seeds() {
+        // Create temporary test folder
+        let temp_path = create_temporary_data_path();
```

Comment:
This  temp path is not needed. The store created in `create_store` already  cleans up the file on drop